saleor/graphql/extensions/mutations.py

```diff
@@ -44,8 +46,12 @@ class Meta:
     @classmethod
     def perform_mutation(cls, root, info, **data):
         plugin_id = data.get("id")
-        input = data.get("input")
-        instance = cls.get_node_or_error(info, plugin_id, only_type=Plugin)
+        data = data.get("input")
         manager = get_extensions_manager()
-        instance = manager.save_plugin_configuration(instance.name, input)
+        plugin = manager.get_plugin(plugin_id)
+        if not plugin:
+            raise ValidationError(
+                {"id": ValidationError("Plugin doesn't exist", code="not_found")}
```

Comment:
Instead of hardcoded `not_found` we shuold use the code from enum, probably from `ExtensionsError` in this case.